src/Compilers/CSharp/Portable/Symbols/AssemblySymbol.cs

```diff
@@ -442,19 +442,16 @@ internal bool RuntimeSupportsStaticAbstractMembersInInterfaces
 
         /// <summary>
         /// Whether the target runtime supports numeric IntPtr types.
-        /// This test hook should be removed once TargetFramework.Net70 is added.
-        /// Tracked by https://github.com/dotnet/roslyn/issues/61235
         /// </summary>
-        internal virtual bool RuntimeSupportsNumericIntPtr
+        internal bool RuntimeSupportsNumericIntPtr
         {
             get
             {
                 // CorLibrary should never be null, but that invariant is broken in some cases for MissingAssemblySymbol.
                 // Tracked by https://github.com/dotnet/roslyn/issues/61262
-                return CorLibrary?.RuntimeSupportsNumericIntPtr == true;
+                return CorLibrary is not null &&
```

Comment:
>CorLibrary is not null && [](http://example.com/codeflow?start=23&length=25)

Is the `CorLibrary` check needed here? It's not included in similar properties below.

Reply:
It is. One test reaches this with a null CorLibrary (`CompilationReferences_More`, crash in `EmitDifference`). This is tracked by above issue.
I'm not sure why numeric IntPtr is uniquely susceptible, but it might be because it's involved in metadata decoding for a type that already exists in the BCL ref assemblies.